tasks/push-snapshot/tests/test-push-snapshot-pushsourcecontainer.yaml

```diff
@@ -96,27 +96,31 @@ spec:
                 exit 1
               fi
 
+              # The sha 570fd5a1e41b5545a24828be5a3df2e85e4a5a67703385f39594837e77e2478e is calculated
```

Comment:
I'm not following this. This string isn't in the mocks.sh file

Reply:
mocks.sh will calculate sha sum of the pull spec string. And for the pull spec used in this test the result happens to be this string. I could also copy/duplicate the commands used in mocks.sh and do the calculation the same way here if you think that's better.

Reply:
`sha256sum` creates this string then?

Reply:
Yes, here in mocks.sh: https://github.com/konflux-ci/release-service-catalog/pull/407/files#diff-9feb1403b29e2fd7aa7291dadd6035be26d9ff05b3a85c831bccb1fd7736caefR40
```sh
      # echo the shasum computed from the pull spec so the task knows if two images are the same
      echo -n "sha256:"
      echo $7 | sha256sum | cut -d ' ' -f 1
```